src/Std/Data/HashMap/Lemmas.lean

```diff
@@ -810,6 +810,8 @@ theorem getThenInsertIfNew?_snd {k : α} {v : β} :
     (getThenInsertIfNew? m k v).2 = m.insertIfNew k v :=
   ext (DHashMap.Const.getThenInsertIfNew?_snd)
 
+theorem mem_of_get_eq {k : α} {v : β} {w} (_ : m.get k w = v) : k ∈ m := w
```

Comment:
We don't use `get` for `HashMap`, we prefer `getElem`, so this lemma shouldn't be necessary. The same is true for a bunch of other files in the PR.